src/other/utils.jl

```diff
@@ -49,14 +49,20 @@ function makeidentifier(s::AbstractString)
     return takebuf_string(res)
 end
 
-function make_unique(names::Vector{Symbol})
+function make_unique(names::Vector{Symbol}; strict=false)
     seen = Set{Symbol}()
     names = copy(names)
     dups = Int[]
     for i in 1:length(names)
         name = names[i]
         in(name, seen) ? push!(dups, i) : push!(seen, name)
     end
+    
+    if strict && length(dups) > 0
+        d = unique(names[dups])
+        throw(ArgumentError("Duplicate variable names: $d"))
```

Comment:
Also mention the existence of the `strict`/`allow_duplicates` argument to help fix the error.